packages/perseus/src/widgets/label-image/answer-pill.tsx

```diff
@@ -74,7 +74,7 @@ export const AnswerPill = (props: {
                     style={[
                         style,
                         popperStyle,
-                        styles.relative,
```

Comment:
I don't think `styles.relative` exists anymore?

Reply:
Thanks for getting rid of that!